censys/common/config.py

```diff
@@ -46,14 +46,11 @@ def get_config() -> configparser.ConfigParser:
     return config
 
 
-def check_config(config: configparser.ConfigParser) -> None:
+def check_config(config: configparser.ConfigParser):
```

Comment:
Shouldn't this guy have a return type hint of None?

Reply:
Yes however to make `darglint` happy I removed it so I do not have to specify a return type in the docstring